app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def new
+
+  end
+
+  def create
+  
+  end
+end
```

Comment:
- [ ] Good job so far 👍 but you have to configure your controller and the corresponding views so that a user would be able to add comments. Kindly add this functionality 💯
![Annotation 2022-06-18 214719](https://user-images.githubusercontent.com/85113567/174456607-0c43b8e3-ba91-4555-a11b-8a6da09a33a7.jpg)